airlock/settings.py

```diff
@@ -165,7 +165,25 @@ def get_env_var(name):
         "ENGINE": "django_sqlite3_backport",
         "NAME": WORK_DIR / "db.sqlite3",
         "OPTIONS": {
-            "init_command": "PRAGMA journal_mode=wal;",
+            # For details on these pragmas see: https://www.sqlite.org/pragma.html
+            "init_command": """
+                /* These settings give much better write performance than the default
+                   without sacrificing consistency guarantees */
+                PRAGMA journal_mode = WAL;
+                PRAGMA synchronous = NORMAL;
+
+                /* Enforce foreign keys which SQLite doesn't do by default only for
+                   backwards compatibility reasons */
+                PRAGMA foreign_keys = ON;
+
+                /* How long (in ms) to let one write transaction wait for another */
+                PRAGMA busy_timeout = 5000;
+
+                /* The default cache size is 2MB but we can afford more! Note negative
+                   values set cache size in KB, positive numbers set it by number of
+                   database pages */
```

Comment:
wat

Reply:
IKR? https://www.sqlite.org/pragma.html#pragma_cache_size

It must be that SQLite originally only supported setting the cache size in terms of number of pages (which is how it is actually implemented internally) and then later, as a convenience, wanted to allow setting it in terms of memory used. And this was the backwards compatibility trick they come up with. It's not _particularly_ lovely, is it?